buildSrc/src/main/java/com/osacky/flank/gradle/YamlConfigWriterTask.kt

```diff
@@ -3,13 +3,20 @@ package com.osacky.flank.gradle
 import org.gradle.api.DefaultTask
 import org.gradle.api.file.ProjectLayout
 import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
+import java.io.File
 import javax.inject.Inject
 
-open class YamlConfigWriterTask @Inject constructor(private val config: FladleConfig, private val extension: FlankGradleExtension, private val projectLayout: ProjectLayout) : DefaultTask() {
+open class YamlConfigWriterTask @Inject constructor(private val config: FladleConfig, private val extension: FlankGradleExtension, projectLayout: ProjectLayout) : DefaultTask() {
 
   private val yamlWriter = YamlWriter()
 
+  private val fladleDir = projectLayout.fladleDir.get().asFile
```

Comment:
Not necessary for this PR, just something to be aware of this will cause overlapping outputs when using multiple configs. I wanted to fix this with: https://github.com/runningcode/fladle/issues/147
We could create a directory based on the config name to solve this.

Reply:
does this need to be annotated with `@Internal`? (check if there are build warnings around this)

Reply:
`@Internal` is not needed because private fields don't generate getters.

I usually make the output configurable, but I'm trying to do the minimum changes to the current behavior or I would make a very big PR.

I like to do something like:
```
@get:OutputDirectory
val outputDir: DirectoryProperty = objectFactory.directoryProperty().convention(layout.buildDirectory.dir("fladle"))
```

Reply:
yeah, i like your approach of making this minimal, we can always fix the other stuff later. thanks!